tests/integration-service/integration.go

```diff
@@ -201,6 +220,21 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 			Expect(f.AsKubeDeveloper.IntegrationController.WaitForAllIntegrationPipelinesToBeFinished(testNamespace, applicationName, snapshot)).To(Succeed())
 		})
 
+		It("checks if the status of integration test is reported in the Snapshot", func() {
```

Comment:
It is better use different names for the two `It`.

Reply:
Why? It's under different When/context

Reply:
When there is some error, I feel we can find it quickly by searching the `It` name

Reply:
updated

for example, also those steps are having the same name in multiple contexts:
* `checks if the Snapshot is created`
* `checks if the BuildPipelineRun is signed`
* `checks if the Build PipelineRun got annotated with Snapshot name`

Reply:
Thanks!